lib/sixpack/configuration.rb

```diff
@@ -1,13 +1,14 @@
 module Sixpack
   class Configuration
-    attr_accessor :base_url, :user, :password
+    attr_accessor :base_url, :user, :password, :read_timeout
 
     def initialize
       @base_url = 'http://localhost:5000'
+      @read_timeout = 1.0
     end
 
     def to_hash
-      config = {base_url: @base_url}
+      config = { base_url: @base_url, read_timeout: @read_timeout }
```

Comment:
Not sure we need these extra spaces.

Reply:
yeah.. following rubocop best practises and it recommended
Space inside { missing. (convention:Layout/SpaceInsideHashLiteralBraces)

Reply:
Huh. TIL